benchmarks/pvc/input_linear.in

```diff
@@ -0,0 +1,198 @@
+PvcGemmBF16BF16FP32_RCR_Linear --bm_name=bf16_bf16_fp32 --l=16 --m=1 --k=512 --n=32768
+PvcGemmBF16BF16FP32_RCR_Linear --bm_name=bf16_bf16_fp32 --l=16 --m=1 --k=16384 --n=7168
+PvcGemmBF16BF16FP32_RCR_Linear --bm_name=bf16_bf16_fp32 --l=16 --m=1 --k=7168 --n=36864
+PvcGemmBF16BF16FP32_RCR_Linear --bm_name=bf16_bf16_fp32 --l=16 --m=1 --k=18432 --n=7168
+PvcGemmBF16BF16FP32_RCR_Linear --bm_name=bf16_bf16_fp32 --l=4096 --m=1 --k=7168 --n=4096
+PvcGemmBF16BF16FP32_RCR_Linear --bm_name=bf16_bf16_fp32 --l=4096 --m=1 --k=2048 --n=7168
+
+PvcGemmBF16BF16FP32_RCR_Linear --bm_name=bf16_bf16_fp32 --l=32 --m=1 --k=7168 --n=1536 
+PvcGemmBF16BF16FP32_RCR_Linear --bm_name=bf16_bf16_fp32 --l=32 --m=1 --k=1536 --n=24576
+PvcGemmBF16BF16FP32_RCR_Linear --bm_name=bf16_bf16_fp32 --l=32 --m=1 --k=7168 --n=576
+PvcGemmBF16BF16FP32_RCR_Linear --bm_name=bf16_bf16_fp32 --l=32 --m=1 --k=512 --n=32768
+PvcGemmBF16BF16FP32_RCR_Linear --bm_name=bf16_bf16_fp32 --l=32 --m=1 --k=16384 --n=7168
+PvcGemmBF16BF16FP32_RCR_Linear --bm_name=bf16_bf16_fp32 --l=32 --m=1 --k=7168 --n=36864
+PvcGemmBF16BF16FP32_RCR_Linear --bm_name=bf16_bf16_fp32 --l=32 --m=1 --k=18432 --n=7168
+PvcGemmBF16BF16FP32_RCR_Linear --bm_name=bf16_bf16_fp32 --l=8192 --m=1 --k=7168 --n=4096
+PvcGemmBF16BF16FP32_RCR_Linear --bm_name=bf16_bf16_fp32 --l=8192 --m=1 --k=2048 --n=7168
+
+PvcGemmBF16BF16FP32_RCR_Linear --bm_name=bf16_bf16_fp32 --l=64 --m=1 --k=7168 --n=1536 
+PvcGemmBF16BF16FP32_RCR_Linear --bm_name=bf16_bf16_fp32 --l=64 --m=1 --k=1536 --n=24576
+PvcGemmBF16BF16FP32_RCR_Linear --bm_name=bf16_bf16_fp32 --l=64 --m=1 --k=7168 --n=576
+PvcGemmBF16BF16FP32_RCR_Linear --bm_name=bf16_bf16_fp32 --l=64 --m=1 --k=512 --n=32768
+PvcGemmBF16BF16FP32_RCR_Linear --bm_name=bf16_bf16_fp32 --l=64 --m=1 --k=16384 --n=7168
+PvcGemmBF16BF16FP32_RCR_Linear --bm_name=bf16_bf16_fp32 --l=64 --m=1 --k=7168 --n=36864
+PvcGemmBF16BF16FP32_RCR_Linear --bm_name=bf16_bf16_fp32 --l=64 --m=1 --k=18432 --n=7168
+PvcGemmBF16BF16FP32_RCR_Linear --bm_name=bf16_bf16_fp32 --l=16384 --m=1 --k=7168 --n=4096
+PvcGemmBF16BF16FP32_RCR_Linear --bm_name=bf16_bf16_fp32 --l=16384 --m=1 --k=2048 --n=7168
+
+PvcGemmBF16BF16FP32_RCR_Linear --bm_name=bf16_bf16_fp32 --l=128 --m=1 --k=7168 --n=1536 
+PvcGemmBF16BF16FP32_RCR_Linear --bm_name=bf16_bf16_fp32 --l=128 --m=1 --k=1536 --n=24576
+PvcGemmBF16BF16FP32_RCR_Linear --bm_name=bf16_bf16_fp32 --l=128 --m=1 --k=7168 --n=576
+PvcGemmBF16BF16FP32_RCR_Linear --bm_name=bf16_bf16_fp32 --l=128 --m=1 --k=512 --n=32768
+PvcGemmBF16BF16FP32_RCR_Linear --bm_name=bf16_bf16_fp32 --l=128 --m=1 --k=16384 --n=7168
+PvcGemmBF16BF16FP32_RCR_Linear --bm_name=bf16_bf16_fp32 --l=128 --m=1 --k=7168 --n=36864
+PvcGemmBF16BF16FP32_RCR_Linear --bm_name=bf16_bf16_fp32 --l=128 --m=1 --k=18432 --n=7168
+PvcGemmBF16BF16FP32_RCR_Linear --bm_name=bf16_bf16_fp32 --l=32768 --m=1 --k=7168 --n=4096
+PvcGemmBF16BF16FP32_RCR_Linear --bm_name=bf16_bf16_fp32 --l=32768 --m=1 --k=2048 --n=7168
```

Comment:
These examples (& all the others with equivalent K N shapes above) are supposed to represent group gemm with 256 experts. Needs to be changed.

Reply:
I've implemented the MoE cases as a regular batched GEMM, with `bs` rows in the A matrix (since our As are actually vectors) and batch size 256 (256 B matrices, one per expert). I believe this is equivalent but I'd like to have that reviewed by someone who understands it better.

Furthermore, the largest size cases fail for the same reasons addressed in https://github.com/codeplaysoftware/cutlass-fork/pull/313. In this case, the issue seems more fundamental (trying to use `cute::cosize(a_large_layout)` overflows before we have the chance to do anything about it. Unsure how to proceed & this is low priority at present.